### ISSUE TITLE
Updating ebuilds

### ISSUE THREAD

**Issue by kpcyrd** [cardoe/cargo-ebuild#16]
It seems `cargo ebuild` is generating a new ebuild every time, it might be a good idea to use the old ebuild as a base

**Comment by cardoe** [comment]
Its going to be quite the project to parse an existing ebuild since its parsed as a shell script. Something a bit easier and more easily achieved would be #10 where the Cargo.toml could have extra metadata fields that would correspond to ebuild fields. This is how 
https://crates.io/crates/cargo-deb and 
https://crates.io/crates/cargo-rpm operate.

**Comment by cardoe** [comment]
Do you have an example of modifications you which you could persist between versions?

**Comment by kpcyrd** [comment]
Yes, all ebuilds in my overlay: https://github.com/kpcyrd/overlay/tree/master/net-analyzer
